visidata/clipboard.py

```diff
@@ -129,8 +129,11 @@ def delete_row(sheet, rowidx):
 
 @Sheet.api
 def paste_after(sheet, rowidx):
-    to_paste = list(deepcopy(r) for r in reversed(vd.memory.cliprows))
-    sheet.addRows(to_paste, index=rowidx)
+    if vd.memory.cliprows:
+        to_paste = list(deepcopy(r) for r in reversed(vd.memory.cliprows))
+        sheet.addRows(to_paste, index=rowidx)
+    else:
+        vd.warning('nothing to paste')
```

Comment:
Would it make sense to change this to

```
if not vd.memory.cliprows:  #1739
     vd.warning('nothing to paste')
     return
  ```
  
  If it does, that would limit the diff to the relevant change, and include a reference to the bug report!

Reply:
Sure thing, done!